app/javascript/components/__tests__/Game.test.js

```diff
@@ -9,6 +9,25 @@ import { MemoryRouter } from 'react-router-dom';
 
 const user = userEvent.setup()
 
+const fakeuser = {
+    user: {
+        name: 'John', email: 'a@a.com', 
+        games: [{ 'sessions': ['a', 'b'] }, { 'sessions': ['a', 'b', 'c', 'd'] }]
+    },
+    setUser: () => { },
+    error: null,
+    setError: () => { },
+    loading: false,
+    setLoading: () => { },
+
+}
```

Comment:
Do you have other tests that need fake users? This seems like great data for a [factory](https://github.com/thoughtbot/factory_bot) or a [fixture](https://api.rubyonrails.org/v3.1/classes/ActiveRecord/Fixtures.html).

Reply:
I haven't really looked into factories/fixtures, but I'll dive into that! It _sounds_ like something that would make my life easier with these tests haha